### ISSUE TITLE
In "dev" branch, bitcoin.conf "generate=1" no longer works

### ISSUE THREAD

**Issue by gandrewstone** [gandrewstone/bitcoin#649]
Error reading configuration file: unrecognized option 'generate'

chain_nol=1
generate=1

**Comment by gandrewstone** [comment]
also:
txretryinterval=2000000                                                                                                                                          
blkretryinterval=2000000

(these were converted to CTweaks, but the old parameter names were left.  Perhaps its time to obsolete them).  But if so, we need a change in UnlimitedSetup...

**Comment by sickpig** [comment]
@gandrewstone confirmed. 
I turned on mining via RPC during the test of PR #642 and it worked. If added to the conf file this is the error I get on startup:

```
Error reading configuration file: unrecognized option 'generate'
```

**Comment by kyuupichan** [comment]
There is no code in the codebase that I can see that asks for "-generate" command line arg.

I do see code for txretryinterval and blkretryinterval so I can add those.

**Comment by sickpig** [comment]
@gandrewstone does this use to work after e4422c90 ?

**Comment by sickpig** [comment]
@gandrewstone putting `generate=1` doesn't work on `release` either. 
The only difference is that now `dev` warns you about the unknown params and exit.

I've checked the code of `release` and it seems there's no code to deal with `generate` on command line and/or conf file.

**Comment by ftrader** [comment]
That's a bit weird though.

I still remember the times when `generate` worked as a cmd line option. But I think that might have been from before BU's split into a 'release' and 'dev' branches. Might even have been on Core.
Long time ago.

I see little value in having it as a config file / command line parameter when one can turn it on with an RPC call.

CPU mining is only useful for testing, and in tests one can easily do the RPC.

**Comment by sickpig** [comment]
Closing due to the fix made by @kyuupichan  on #651 and because `generate` didn't work in the config file since a long time (didn't even work on a local copy I have of the `0.12.1bu` branch)
